### DISCUSSION TITLE
What exactly does pyenv insert into your PATH: `$PYENV_ROOT/shims` or `$PYENV_ROOT/bin`?

### DISCUSSION THREAD

**Discussion by lackhoa** [pyenv/pyenv#2030] [Q&A] [answered]
In the latest README, there's this paragraph
> pyenv works by inserting a directory of shims at the front of your PATH:
`$(pyenv root)/shims:/usr/local/bin:/usr/bin:/bin`

But then the configuration step says this
> `export PATH="$PYENV_ROOT/bin:$PATH"`

I installed pyenv on MacOS and this `$PYENV_ROOT/bin` doesn't exist, only `$PYENV_ROOT/shims`. I know the doc says that I don't need to modify PATH on Mac, but then pyenv doesn't work because my PATH isn't modified.

What's going on: is the doc wrong or is is it something wrong with my installation?

The version is `2.0.4`, installed from homebrew.

**Comment by native-api** [comment, marked as answer]
https://github.com/pyenv/pyenv#advanced-configuration:

> Step 1 is done by eval "$(pyenv init --path)", the others are done by eval "$(pyenv init -)".
> 
>   1. Sets up your shims path. This is the only requirement for pyenv to function properly. You can do this by hand by prepending $(pyenv root)/shims to your $PATH.
>

**Reply by lackhoa** [reply]
I mistooked the `eval "$(pyenv init --path)"` part with `eval "$(pyenv init -)"`. Thanks!
